vllm/envs.py

```diff
@@ -250,6 +251,10 @@ def get_default_config_root():
     "VLLM_ATTENTION_BACKEND":
     lambda: os.getenv("VLLM_ATTENTION_BACKEND", None),
 
+    # If set, vllm will not use flashinfer sampler
+    "VLLM_NO_FLASHINFER_SAMPLER":
```

Comment:
what's the consideration for using it or not?

can we use it as long as flashinfer is used?

we have too many flags now.

Reply:
I would suggest keep this flag at least for now. The FlashInfer sampling kernel is not stable yet, so SGLang also provides a flag to disable it so that users can be unblocked once there's an issue.

Reply:
got it, thanks.